ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -984,7 +984,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=node \n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "export NODE_OPTIONS=--openssl-legacy-provider\nexport BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=node \n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `export NODE_OPTIONS=--openssl-legacy-provider` is a workaround for the OpenSSL 3.0 compatibility issue with Node.js. This should be fine as long as you're aware that it forces the use of the legacy provider and could potentially have security implications if not updated when a fix is available.
```diff
- export NODE_OPTIONS=--openssl-legacy-provider\nexport BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=node \n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+ export BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=node \n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->